app/assets/stylesheets/form/_slider.scss

```diff
@@ -0,0 +1,298 @@
+/*! nouislider - 14.7.0 - 4/6/2021 */
+/* Functional styling;
+ * These styles are required for noUiSlider to function.
+ * You don't need to change these rules to apply your design.
+ */
+ .noUi-target,
+ .noUi-target * {
+   -webkit-touch-callout: none;
```

Comment:
I know these are just copy pasted from their thingie, but let's remove browser prefixes, those get added automatically anyway (and most are incredibly outdated)